docs/COLLABORATION.md

```diff
@@ -0,0 +1,35 @@
+# Collaboration
+
+## Principles
+
+### Voluntary Work
+
+Organic Maps welcomes all volunteer contributors from any domain. Volunteer Contributors work at their own pace. Please be prepared for volunteer work to be carried out in a more spontaneous or unstructured manner.
+
+### Funded Work
+
+Organic Maps obtains donations and grant funding from public institutions to support new feature development. Please expect such work to be structured as projects with clear scope, deadlines, assigned teams, and a person in charge.
+
+### Interactivity
+
+The objective is to merge all contributions as soon as they are ready and meet the merge criteria. However, certain changes may require expertise from individuals who are unavailable or busy with other tasks that have deadlines. Please be patient and collaborate on the provided comments to ensure all required criteria are met.
+
+### Priorities
+
+The roadmap is more of a collection of funded or expected-to-be-funded projects. Voluntary work is difficult to plan, and features and bug fixes may arise out of order. The team will make every effort to organize work in key areas, completing all preliminary product and UI/UX tasks to ensure they are ready for execution by volunteers.
+
+### Publicity
+
+All development communication should occur through public channels, avoiding direct messages and private chats whenever possible. Any decision made must be documented in writing within the ticket, ensuring that the motivation and reasoning are clear and understandable to all.
+
+### Artifacts
+
+Collaboration should be conducted in a way that produces tangible artifacts, such as tickets, design mockups, pull requests, and other relevant documentation. Chatting in messaging platforms (e.g., Telegram) often leads to time wastage and does not generate artifacts. Please respect your time and the time of others by prioritizing the creation of artifacts over generating chat logs.
```

Comment:
CONTRIBUTING.md looks solid already... This paragraph is more like a guideline. What kind of guideline we can create for such documentation?

Reply:
Yeap its quite big already so ideas are welcome how to decompose it :))

Maybe separate bug reporting / feature requests into another doc.

Move legal stuff (DCO) also...

One of the (raw) ideas is also to move some team-specific docs into the TEAMS.md, so that e.g. people who'd like to contribute to the Android app could find everything in one place (including an Android-specific INSTALL doc)..